docs/developers/index.md

```diff
@@ -206,6 +206,7 @@ The Ethereum network is made up of many nodes who run compatible client software
 
 ### Looking for other options?
 - [Ethereum Developer Tools List #Patterns—best-practices](https://github.com/ConsenSys/ethereum-developer-tools-list#patterns--best-practices)
+- [Aggregated collection of security recommendations and best practices (Updated on 21-5-2019)](https://github.com/guylando/KnowledgeLists/blob/master/EthereumSmartContracts.md)
```

Comment:
Do the other links have a time stamp?

Reply:
Also, why not put this into the `Smart Contract Security Best Practices Guide` subsection?

Reply:
1. Well as I wrote in the pull message above, I think it is important to let people be aware in case they are reading old information. During my research I have been reading too much things which turned out to be irrelevant after solidity 0.5.0 changes.
2. Where else to put it? That is exactly what the link includes in my opinion

Reply:
Sure, your github repo should contain a date, but would you update this repo every time you make a change?

You put into the subsection `Looking for other options?`. I propose to instead move it up one subsection into `Smart Contract Security Best Practices Guide `.

Reply:
You are right, I was looking at it from another perspective of letting people know that this information is fresh as of may 2019 compared to some of the others but maybe it ruins more than it helps because of what you said. Will remove the timestamp.

Will Do.

Reply:
fixed